README.md

```diff
@@ -91,17 +91,17 @@ module.exports = defineConfig({
 npx cypress run
 ```
 
-2. If there are failed tests, run the following command from the **directory of the project's `cypress.config`**:
+2. If there are failed tests, run the following command from the **failedTestDirectory of the project**:
 
 ```bash
-npx cypress-last-failed run
+npx cypress-plugin-last-failed run
```

Comment:
The name change was because when i execute it like in the README I got this npm error and only with the updated name it's solved
![imagen](https://github.com/dennisbergevin/cypress-plugin-last-failed/assets/55634266/e20b6d90-5988-4127-985f-b9dc8b5e3979)

Reply:
Thanks for pointing this discrepancy out!